### PR TITLE
Astronomer Software 0.30.1 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

**Review by cmarteepants — APPROVED** [review]

**Review by kwcanuck — APPROVED** [review]

File: software/release-notes.md [hunk 1]
```diff
@@ -9,6 +9,24 @@ description: Astronomer Software release notes.
 
 0.30 is the latest stable and long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
 
+## 0.31.0
```
Comment (vishwas-astro) [inline software/release-notes.md:12]: 0.30.1, not 0.31.0

File: software/release-notes.md [hunk 2]
```diff
@@ -9,6 +9,24 @@ description: Astronomer Software release notes.
...
+
+Release date: September 12, 2022
+
+### Additional improvements
+
+- You can now configure a custom `scrape_interval` for Prometheus using the `--set` flag when [applying a platform config change with Helm](apply-platform-config.md).
```
Comment (vishwas-astro) [inline software/release-notes.md:18]: This change was reverted. Does not need to go into release notes. cc @danielhoherd @cmarteepants

**Review by vishwas-astro — CHANGES_REQUESTED** [review]

File: software/release-notes.md (hunk 2 above)
Comment (danielhoherd) [reply software/release-notes.md:18]: That's correct. This feature was broken, so it was reverted. It is planned for a future release though so hold onto these sections for future use.

File: software/release-notes.md (hunk 1 above)
Comment (cmarteepants) [reply software/release-notes.md:12]: Good catch! I missed that one

File: software/release-notes.md (hunk 2 above)
Comment (jwitz) [inline software/release-notes.md:18]:
```suggestion
```

File: software/release-notes.md [hunk 3]
```diff
@@ -9,6 +9,21 @@ description: Astronomer Software release notes.
 
 0.30 is the latest stable and long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
 
+## 0.31.0
+
+Release date: September 12, 2022
+
```
Comment (jwitz) [inline software/release-notes.md:15]:
```suggestion
```

File: software/release-notes.md [hunk 4]
```diff
@@ -9,6 +9,21 @@ description: Astronomer Software release notes.
 
 0.30 is the latest stable and long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
 
+## 0.31.0
```
Comment (jwitz) [inline software/release-notes.md:12]:
```suggestion
## 0.30.1
```

**Pull request merged**